bundle/phases/deploy.go

```diff
@@ -169,22 +175,6 @@ func deployPrepare(ctx context.Context, b *bundle.Bundle) {
 		libraries.CheckForSameNameLibraries(),
 		// SwitchToPatchedWheels must be run after ExpandGlobReferences and after build phase because it Artifact.Source and Artifact.Patched populated
 		libraries.SwitchToPatchedWheels(),
-
-		// libraries.Upload() not just uploads but also replaces local paths with remote paths.
-		// TransformWheelTask depends on it and planning also depends on it.
-		libraries.Upload(),
-		trampoline.TransformWheelTask(),
-
-		mutator.ResolveVariableReferencesOnlyResources(
-			"resources",
-		),
-
-		// Reads (typed): resources.pipelines.*.{catalog,schema,target}, resources.volumes.*.{catalog_name,schema_name} (checks for schema references)
-		// Updates (typed): resources.pipelines.*.{schema,target}, resources.volumes.*.schema_name (converts implicit schema references to explicit ${resources.schemas.<schema_key>.name} syntax)
-		// Translates implicit schema references in DLT pipelines or UC Volumes to explicit syntax to capture dependencies
-		// Needs to be run after ${resources} resolution since otherwise that undoes the change here.
-		// TODO: one we have depends_on support we should leverage that here and move this back to initialize phase.
-		resourcemutator.CaptureSchemaDependency(),
```

Comment:
I've reverted this here https://github.com/databricks/cli/pull/3443 

I suggest we land revert first and rebase, would be easier to review.